monkey/infection_monkey/network/fingerprinter_manager.py

```diff
@@ -0,0 +1,14 @@
+import logging
+from typing import Sequence
+from infection_monkey.utils.load_plugins import get_instances
+from infection_monkey.network.HostFinger import HostFinger
+
+LOG = logging.getLogger(__name__)
+
+
+def get_fingerprint_instances() -> Sequence[HostFinger]:
+    """
+    Returns the fingerprint objects according to configuration as a list
+    :return: A list of HostFinger objects.
+    """
+    return get_instances(__package__, __file__, HostFinger)
```

Comment:
Perhaps "full import path" is prudent here, in case this file will be moved to another location? 
Otherwise, document that this function must stay next to the fingerprinters.